src/components/steps/WalletEncryptionPassword.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <NavigationCard
+    class="wallet-encryption"
+    title="Encrypt your wallet with a password"
+    previousText="Back"
+    nextText="Next"
+    :previousStep="previousStep"
+    :nextStep="nextStep"
+  >
+    <div class="content">
+      <p class="paragraph">
+        <strong>PLEASE NOTE:</strong> this password encrypts your Witnet wallet only on this
+        computer. This is not your backup and you cannot restore your wallet with this password.
+        Your seed phrase is still your ultimate backup.
+      </p>
+      <div class="form-row">
+        <label class="label">Password</label>
+        <input class="password-input" type="password" v-model="password">
```

Comment:
I think we should use 'element' for the input and add ours in the following issues